spmd/tensor/ops/matrix_ops.py

```diff
@@ -37,19 +38,28 @@ def mm_prop(
     ):
         return mat1_spec
     else:
+        # TODO(anj): Add logging (or something else) to indicate no output sharding
+        # spec specified. If we don't have autoboxing rules specified in the op below,
+        # we will throw an error without a populated `failed_reason`.
         # not local compute, need to rely on auto redistribute, return None
         return None
 
 
-@register_prop_rule("aten.mm.default")
 def mm_rules(op_schema: OpSchema) -> OutputSharding:
-    mat1_spec, mat2_spec = op_schema.args_spec
+    mat1_spec, mat2_spec = op_schema.args_schema
     return OutputSharding(mm_prop(mat1_spec, mat2_spec))
 
 
+default_mm_ops = ["aten.mm.default", "aten.mul.Tensor"]
```

Comment:
why we are changing matrix ops for torch.mul? `torch.mul` is not a matrix op, we shouldn't change this file.

Reply:
Clarified offline: Mistook the doc example that had a MM output. Updated to reflect pointwise.